code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -412,12 +412,26 @@
 
 /obj/item/weapon/robot_module/combat/New()
 	src.modules += new /obj/item/device/flash/cyborg(src)
+	src.modules += new /obj/item/weapon/restraints/handcuffs/cable/zipties/cyborg(src)
 	src.modules += new /obj/item/borg/sight/thermal(src)
-	src.modules += new /obj/item/weapon/gun/energy/laser/cyborg(src)
+	src.modules += new /obj/item/weapon/gun/energy/gun/cyborg(src)
+	src.modules += new /obj/item/weapon/gun/energy/plasmacutter(src)

```

Comment:
Plasma cutter is mostly useless for pressurized combat, and borgs can't recharge theirs. If you want to give them a way to dig through walls, try a sonic jackhammer or drill.